src/openzeppelin/introspection/erc165/library.cairo

```diff
@@ -10,33 +10,29 @@ from starkware.cairo.common.bool import TRUE
 from openzeppelin.utils.constants.library import INVALID_ID, IERC165_ID
 
 @storage_var
-func ERC165_supported_interfaces(interface_id: felt) -> (is_supported: felt):
-end
+func ERC165_supported_interfaces(interface_id: felt) -> (is_supported: felt) {
+}
 
-namespace ERC165:
-    func supports_interface{
-            syscall_ptr: felt*,
-            pedersen_ptr: HashBuiltin*,
-            range_check_ptr
-        } (interface_id: felt) -> (success: felt):
-        if interface_id == IERC165_ID:
-            return (TRUE)
-        end
+namespace ERC165 {
+    func supports_interface{syscall_ptr: felt*, pedersen_ptr: HashBuiltin*, range_check_ptr}(
+        interface_id: felt
+    ) -> (success: felt) {
+        if (interface_id == IERC165_ID) {
+            return (success=TRUE);
+        }
```

Comment:
```suggestion
        interface_id: felt
    ) -> (felt) {
        if (interface_id == IERC165_ID) {
            return (TRUE);
        }
```
not sure if we want to do the former or this. not sure if verbosity in here is a win or a loss